lighthouse-core/report/report-generator.js

```diff
@@ -263,18 +274,13 @@ class ReportGenerator {
   }
 
   /**
-   * Generates the Lighthouse report HTML.
-   * @param {!Object} results Lighthouse results.
-   * @param {!string} reportContext What app is requesting the report (eg. devtools, extension)
-   * @return {string} HTML of the report page.
```

Comment:
Actually not a big change here. Just moving formatters registration to it's own method. So I can inject some code by overriding this method.